subworkflows/nf-core/gatk4/tumor_normal_somatic_variant_calling/main.nf

```diff
@@ -121,15 +142,29 @@ workflow GATK_TUMOR_NORMAL_SOMATIC_VARIANT_CALLING {
     GETPILEUPSUMMARIES_TUMOR ( pileup.tumor.map{
                                     meta, cram, crai, intervals ->
 
-                                    [[patient:meta.patient, normal_id:meta.normal_id, tumor_id:meta.tumor_id, sex:meta.sex, id:meta.tumor_id, num_intervals:meta.num_intervals],
+                                    [[
+                                        id:             meta.tumor_id,
+                                        normal_id:      meta.normal_id,
+                                        num_intervals:  meta.num_intervals,
+                                        patient:        meta.patient,
+                                        sex:            meta.sex,
+                                        tumor_id:       meta.tumor_id,
+                                    ],
                                         cram, crai, intervals]
                                 },
                                 fasta, fai, dict, germline_resource_pileup, germline_resource_pileup_tbi )
 
     GETPILEUPSUMMARIES_NORMAL ( pileup.normal.map{
                                     meta, cram, crai, intervals ->
 
-                                    [[patient:meta.patient, normal_id:meta.normal_id, tumor_id:meta.tumor_id, sex:meta.sex, id:meta.normal_id, num_intervals:meta.num_intervals],
+                                    [[
+                                        id:             meta.normal_id,
+                                        normal_id:      meta.normal_id,
+                                        num_intervals:  meta.num_intervals,
+                                        patient:        meta.patient,
+                                        tumor_id:       meta.tumor_id,
+                                        sex:            meta.sex,
```

Comment:
Probably not a problem in a meta map, but all others have `tumor_id` as last elem

Reply:
good point, we could sort all meta map alphabetically, that way, we don't have to think about it anymore

Reply:
tried doing that, clearly missed a few. I'll fix the ones you found now :)